tests/ui/push-health/TestFailure_test.jsx

```diff
@@ -24,6 +24,7 @@ beforeEach(() => {
     },
   });
   setUrlParam('repo', repoName);
+  failure.key = 'wazzon';
```

Comment:
wazzon?

Reply:
It's a slang term for "What's on TV?" from a comic called "Life in Hell" by Matt Goening (before he made the Simpsons).  Soo...  that's a pretty un-popped pop-culture reference...  For some reason it always stuck in my head...  :)